File: fizzbuzz.dart
void main() {
  fizz(n) => n % 3 ==  0 ? "Fizz" : "";
  buzz(n) => n % 5 == 0 ? "Buzz" : "";
  number(n) => n % 3 == 0 || n % 5 == 0 ? "" : n;
  for (num i = 0; i++ < 100;) {
    print( "${fizz(i)}${buzz(i)} ${number(i)}" );
  }
}

File: rust/fizzbuzz.rs
fn main(){
	let mut count = 0u32;
	
	loop{

		count += 1;
		if count % 3 == 0 {
			print!("Fizz");
		}
		if count % 5 == 0 {
			print!("Buzz");
		}
		if (count % 3 != 00 && count % 5 != 0) {
			print!("{}", count);
		}
		println!("");
		if (count == 100) {
			break;
		}
	}
}